include/avt_vimba_camera/avt_vimba_camera.h

```diff
@@ -57,6 +57,7 @@ enum FrameStartTriggerMode
   Freerun,
   FixedRate,
   Software,
+  SyncIn0,
```

Comment:
http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Renum-class
Prefer class enums over “plain” enums

Reply:
@icolwell-as is the Trigger enum necessary? Maybe we could just check if the received feature is in the array?

Reply:
Let's address it with another PR later on.